src/core_atmosphere/Registry.xml

```diff
@@ -4716,5 +4716,6 @@
 
 #ifdef DO_PHYSICS
 #include "physics/Registry_noahmp.xml"
+#include "registry.chem.xml"
 #endif
```

Comment:
I'm wondering if there should be a "chem" directory added under src/core_atmosphere(?). I have doubt this will be accepted by NCAR, knowing that they have plans with CHEMPAS, but I'm not sure how they are connecting it up. An alternative might be to just have a "smoke" registry (if a separate registry is really needed) and keep it in the physics directory. Not sure what you all think...

Reply:
Thanks for pointing this out. We will rename it as "registry.smoke.xml" to be consistent with the code in physics/physics_noaa/SMOKE.

Reply:
Thanks Joe and Clark. registry.chemistry.xml is moved into the physics directory.